R/pkg/NAMESPACE

```diff
@@ -265,6 +265,7 @@ exportMethods("%in%",
               "var_samp",
               "weekofyear",
               "when",
+              "window",
```

Comment:
this should be added to https://github.com/apache/spark/blob/master/R/pkg/R/generics.R?